src/temperature_component.cpp

```diff
@@ -97,11 +107,6 @@ void TemperatureComponent::init( Core* coreptr ) {
     tgav_constrain.allowInterp( true );
     tgav_constrain.name = D_TGAV_CONSTRAIN;
 
-    // Define the doeclim parameters
```

Comment:
@bpbond because these parameters will be read in via the ini file setting them here is seems redundant, okay to delete or is there some reason to keep these lines of code?

Reply:
OK to delete afaik